.gitignore

```diff
@@ -4,6 +4,7 @@ syntax: glob
 
 # Tool Runtime Dir
 /[Tt]ools/
```

Comment:
I thought it might be nice to wait a bit while this transition is in place however I don't feel too strongly about it and can remove it now if we think that's the better choice.